### PR TITLE
Allow mpTelemetry-2.0 to work with Java 8

### PR DESCRIPTION
**Pull request by tevans78**
Allow mpTelemetry-2.0 to work with Java 8

### CODE REVIEW

**Comment by tevans78** [comment]
#build

File: dev/io.openliberty.microprofile.telemetry.1.0.internal.container_fat/fat/src/io/openliberty/microprofile/telemetry/internal/apps/agent/AgentTestResource.java [hunk 1]
```diff
@@ -14,12 +14,13 @@
```
Comment (benjamin-confino) [inline dev/io.openliberty.microprofile.telemetry.1.0.internal.container_fat/fat/src/io/openliberty/microprofile/telemetry/internal/apps/agent/AgentTestResource.java:14]: Copyright is outdated in this file

File: dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/apps/spi/propagator/TestPropagator.java [hunk 2]
```diff
@@ -4,7 +4,7 @@
  * are made available under the terms of the Eclipse Public License 2.0
```
Comment (benjamin-confino) [inline dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/apps/spi/propagator/TestPropagator.java:4]: Copyrights in this file needs updating.

**Comment by benjamin-confino** [comment]
This looks good, I'm happy to approve as soon as its tested. There's a few other files where the copyright wasn't changed but I think those changes are small enough that they don't need updates.

**Comment by tevans78** [comment]
New PB: https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_yvBhUEKbEe-F88sHKWQ5jw

Java 8 SOE: https://libh-proxy1.fyre.ibm.com/cognitive-dev/pipelineAnalysis.html?pipelineId=3d739ba7-06b5-47c6-b047-598eb20d7e13

**Comment by tevans78** [comment]
New PB: https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_x1obAETnEe-ngPvsOTExmw

New Java 8 SOE: https://libh-proxy1.fyre.ibm.com/cognitive/pipelineAnalysis.html?pipelineId=d7019b25-821d-4e29-bc3b-fc2da45953da

**Comment by Azquelt** [comment]
#build
#spawn.fullfat.buckets=io.openliberty.microprofile.telemetry.1.0.internal.container_fat,io.openliberty.microprofile.telemetry.1.0.internal_fat,io.openliberty.microprofile.telemetry.1.0.internal_fat_tck,io.openliberty.microprofile.telemetry.1.1.internal_fat_tck,io.openliberty.microprofile.telemetry.2.0.internal_fat_tck

**Comment by Azquelt** [comment]
New Java 8 SOE: https://libh-proxy1.fyre.ibm.com/cognitive/pipelineAnalysis.html?pipelineId=336da277-1bf7-4f91-9422-79a136e9afa1

**Comment by Azquelt** [comment]
From the failures in the Java8 SOE, it looks like
- `Agent210Test.testHttpClient` is failing because in `AgentTestResource.callHttpClient`, it's receiving `OK\n` when it's expecting `OK`.
  - This isn't a difference we care about for this test so we should `trim()` the response before comparing
- `AgentConfigMultiAppTest.testAgentMultiApp` is failing for repeat `_EE9_FEATURES_MPTEL20_MicroProfile_50`
  - Based on the comment on the test method, this test is unreliable on MP 5.0, so the newly added MP 5.0 repeat IDs should be added to the `@SkipForRepeat` annotation.

**Comment by Yas-Franco** [comment]
Replaced by [29191](https://github.com/OpenLiberty/open-liberty/pull/29191) which is now merged

**Pull request closed without merging**